.azure-pipelines/bootstrap.py

```diff
@@ -671,12 +671,6 @@ def update_sources(options):
             "dials/cbflib",
         )
     }
-    repositories["cctbx_project"] = {
-        "base-repository": "cctbx/cctbx_project",
-        "effective-repository": "dials/cctbx",
-        "branch-remote": "master",
-        "branch-local": "stable",
-    }
```

Comment:
this needs to be left in, otherwise it won't know what to check out

Reply:
or maybe I have this wrong. Ignore pls.

Reply:
Yeah, it first creates the entry then overwrites it!

Reply:
but it used to be harmless to leave in. Does it have to go because we now need a branch from the upstream remo?